Sources/Controllers/2.0/ConfigureScreen/OAConfigureMenuMainScreen.mm

```diff
@@ -286,26 +292,19 @@ - (UITableViewCell *) tableView:(UITableView *)tableView cellForRowAtIndexPath:(
 
 - (void) updateSettingSwitchCell:(OASettingSwitchCell *)cell data:(NSDictionary *)data
 {
-    UIImage *img = nil;
+    cell.imgView.image = nil;
```

Comment:
This line seems unnecessay. You set the image in the 299-th line, so no need to set it no nil here.